setup.py

```diff
@@ -149,6 +149,9 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Framework :: Django',
+        'Framework :: Django :: 1.9',
```

Comment:
1.9 is not supported, better do not include it, instead you can add 1.8, which is supported